sdk/core/perf/test/inc/azure/perf/test/delay_test.hpp

```diff
@@ -56,7 +56,7 @@ namespace Azure { namespace Perf { namespace Test {
     {
       // Increment the counter and fetch the value, then remove 1 to get the previous-increment
       // value
-      auto instanceCount = _detail::DelayTestInstanceCount.fetch_add(1) - 1;
```

Comment:
I'm a bit confused here. The comment says "increment the counter", but the documentation for `std::atomic::fetch_add` says that it adds the first parameter to the value, in which case the new code is adding 0 to the `DelayInstanceCount`.

Reply:
whatever the documentation says it is wrong, under the debugger the fetc_add 1 return 0

Reply:
and really the point of the test is to do a sleep , not to waste time with ftch add